client/views/room/MessageList/components/MessageReactionsList.tsx

```diff
@@ -0,0 +1,35 @@
+import { MessageReactions, MessageReactionAction } from '@rocket.chat/fuselage';
+import React, { ReactElement } from 'react';
+
+import { IMessage } from '../../../../../definition/IMessage';
+import { useOpenEmojiPicker, useReactionsFilter, useReactToMessage, useUserHasReacted } from '../contexts/MessageListContext';
+import { MessageReaction } from './MessageReaction';
+
+const MessageReactionsList = ({ message }: { message: IMessage }): ReactElement | null => {
+	const hasReacted = useUserHasReacted(message);
+	const reactToMessage = useReactToMessage(message);
+	const filterReactions = useReactionsFilter(message);
+	const openEmojiPicker = useOpenEmojiPicker(message);
+
+	if (!message.reactions || !Object.keys(message.reactions).length) {
+		return null;
+	}
```

Comment:
This conditional should be on the parent component.

Message.tsx
```
	{Object.keys(message.reactions).length && <MessageReactionsList />}
```

This way the component will not be loaded.